src/css.js

```diff
@@ -246,7 +246,9 @@ jQuery.extend( {
 			}
 
 			// If a number was passed in, add the unit (except for certain CSS properties)
-			if ( type === "number" ) {
+			// The isCustomProp check can be removed in jQuery 4.0 when we only auto-append
```

Comment:
Is there a url we can reference for this statement/plan?

Reply:
Once this PR lands I'll rebase #4055 and add a removal of this part there so I just need to remember about it until that moment. :)

Reply:
I landed this PR & rebased PR #4055, reverting this fix there.